app/Resources/views/macros/forms.html.twig

```diff
@@ -81,7 +81,7 @@
             <span class="text-muted">({{ msg("optional") }})</span>
         {% endif %}
     </label>
-    <input type="date" class="form-control" name="{{ type }}" placeholder="2016-11-01"{% if required %} required{% endif %} />
+    <input type="date" class="form-control" name="{{ type }}" placeholder="YYYY-MM-DD"{% if required %} required{% endif %} />
```

Comment:
Also mind you this generally isn't a problem because modern browsers will show a datepicker for `<input type='date' />`